adb_graphics/default_specs.yml

```diff
@@ -221,6 +221,12 @@ ceilexp2: # Ceiling - experimental no.2
     <<: *ceil
     ncl_name: CEIL_P0_L2_{grid}
     title: Ceiling (exp-2)
+cfrzr: # Categorical Freezing Rain
+  sfc:
+    ncl_name: CFRZR_P0_L1_GLC0
+cicep: # Categorical Ice Pellets
+  sfc:
+    ncl_name: CFRZR_P0_L1_GLC0
```

Comment:
It would be helpful for consistency to add the "grid" template into each of the ncl_name entries you've added here and below.